pgdrive/envs/multi_agent_pgdrive.py

```diff
@@ -209,30 +209,34 @@ def _after_lazy_init(self):
             self.main_camera.camera_x += self.config["top_down_camera_initial_x"]
             self.main_camera.camera_y += self.config["top_down_camera_initial_y"]
 
-    def _respawn_vehicles(self):
+    def _respawn_vehicles(self, randomize_position=False):
         new_obs_dict = {}
+        if len(self._agent_manager.pending_objects) == 0:
```

Comment:
Do we have a better way to retrieve this information?

Reply:
ok, I will add an interface to return flag only